samples/modules/canopennode/sample.yaml

```diff
@@ -29,4 +29,7 @@ tests:
       SB_CONFIG_BOOTLOADER_MCUBOOT=y
 
   sample.modules.canopennode.no_storage:
+    filter: not CONFIG_BOOTLOADER_MCUBOOT
     extra_args: CONF_FILE=prj_no_storage.conf
+    platform_exclude:
+      - frdm_mcxn947/mcxn947/cpu0/qspi
```

Comment:
This change does not match the Git commit message. Please explain why this is needed.

Reply:
if use no_storage conf, the frdm_mcxn947/mcxn947/cpu0/qspi can not support this test instance. as it is runs from qspi.